.cirrus.yml

```diff
@@ -2,7 +2,7 @@ container:
     image: l.gcr.io/google/bazel:latest
 task:
     name: Build the example document
-    build_script: bazel build //example:my_report
+    build_script: bazel build //example/... --verbose_failures
```

Comment:
Out of curiosity, did you find this helped? If not, consider removing.

Reply:
No, I guess it didn't.
I was mostly the more verbose error message in the python code that helped,
I can remove it.

Reply:
done